classic-azure-ts-vm-provisioners/package.json

```diff
@@ -3,8 +3,8 @@
     "version": "0.1.0",
     "main": "index.js",
     "dependencies": {
-        "@pulumi/azure": "^4.0.0",
-        "@pulumi/command": "^0.0.3",
+        "@pulumi/azure": "^5.0.0",
+        "@pulumi/command": "1.0.0-beta.1",
```

Comment:
Looks like this example may not be covered by tests in misc/tests?  Is it worth adding it there to make sure these changes work and that we don’t regress in the future?

Reply:
Filed #1658 so we don't lose track of it and I can merge this PR.